packages/integrations/vercel/README.md

```diff
@@ -13,9 +39,27 @@ export default defineConfig({
 });
 ```
 
-When you build your project, Astro will know to use the `.vercel/output` folder format that Vercel expects.
+### Targets
+
+You can deploy to different targes:
+
+- `edge`: SSR inside a [Edge function](https://vercel.com/docs/concepts/functions/edge-functions).
+- `serverless`: SSR inside a [Node.js function](https://vercel.com/docs/concepts/functions/serverless-functions).
+- `static`: generates a static website following Vercel's output formats, redirects, etc.
+
+> **Note**: deploying to the Edge has [its limitations](https://vercel.com/docs/concepts/functions/edge-functions#known-limitations) — they can't be more than 1 MB in size and they don't support native Node.js APIs, among others.
+
+You can change where to target by changing the import:
+
+```js
+import vercel from '@astrojs/vercel/edge';
+import vercel from '@astrojs/vercel/serverless';
+import vercel from '@astrojs/vercel/static';
+```
+
+## Usage
 
-## Deploying
+[Read the full deployment guide here.](https://docs.astro.build/en/guides/deploy/vercel)
```

Comment:
Same suggestions as above — highlight this link a little to help drive people to the docs?

```suggestion
📚 **[Read the full deployment guide here.](https://docs.astro.build/en/guides/deploy/vercel)**
```